Enable CONFIG_I2C_MUX_PCA954x, CONFIG_I2C_MUX_PCA954X_DESELECT_ON_EXIT, 
       CONFIG_GPIO_PCA953X
From: wadelnn <wadelnn@users.noreply.github.com>

---
 debian/build/build_amd64_none_amd64/.config | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
index b22d55f..3cde51e 100644
--- a/debian/build/build_amd64_none_amd64/.config
+++ b/debian/build/build_amd64_none_amd64/.config
@@ -3146,7 +3146,8 @@ CONFIG_I2C_MUX=m
 #
 CONFIG_I2C_MUX_GPIO=m
 # CONFIG_I2C_MUX_PCA9541 is not set
-# CONFIG_I2C_MUX_PCA954x is not set
+CONFIG_I2C_MUX_PCA954x=m
+CONFIG_I2C_MUX_PCA954X_DESELECT_ON_EXIT=y
 # CONFIG_I2C_MUX_PINCTRL is not set
 CONFIG_I2C_HELPER_AUTO=y
 CONFIG_I2C_SMBUS=m
@@ -3190,7 +3191,7 @@ CONFIG_I2C_SCMI=m
 CONFIG_I2C_DESIGNWARE_CORE=m
 CONFIG_I2C_DESIGNWARE_PLATFORM=m
 CONFIG_I2C_DESIGNWARE_PCI=m
-# CONFIG_I2C_GPIO is not set
+CONFIG_I2C_GPIO=m
 CONFIG_I2C_KEMPLD=m
 CONFIG_I2C_OCORES=m
 CONFIG_I2C_PCA_PLATFORM=m
@@ -3304,7 +3305,7 @@ CONFIG_GPIO_SCH=m
 #
 # CONFIG_GPIO_MAX7300 is not set
 # CONFIG_GPIO_MAX732X is not set
-# CONFIG_GPIO_PCA953X is not set
+CONFIG_GPIO_PCA953X=m
 # CONFIG_GPIO_PCF857X is not set
 # CONFIG_GPIO_ADP5588 is not set
 
-- 
2.1.4

